Terminal.Gui/Core/View.cs

```diff
@@ -142,6 +142,12 @@ internal enum Direction {
 		View focused = null;
 		Direction focusDirection;
 
+		/// <summary>
+		/// This event is fired once when the application is first loaded. The dimensions of the
+		/// terminal are provided.
+		/// </summary>
+		public Action<Application.ResizedEventArgs> Loaded;
```

Comment:
For simple apps this concept is OK.

However, for anything dynamic, having resized be tied to `Application` loading makes no sense and makes this API coutner-productive. As I noted in another thread, this will never get called if the View is created AFTER the app has been running. This is one of the reasons I wrote `AllViewsTester`, to illustrate bugs in Views that depended on the idea they were all created before `Application.Run` was called.

I do not think we should extend `View` to have a `Loaded` event.

Reply:
You are right @tig. I anticipated and did not reflect on it well. I will present another PR that will only invoke an event once when the view is added to the toplevel. I will close this PR next.